packages/gatsby-source-wordpress/src/steps/create-schema-customization/transform-fields/field-transformers.js

```diff
@@ -40,6 +42,24 @@ export const fieldTransformers = [
     },
   },
 
+  {
+    description: `Lists of Gatsby node interfaces`,
+    test: field => {
+      const implementsNodeInterface = store
+        .getState()
+        .remoteSchema.typeMap.get(findNamedTypeName(field.type))
+        ?.interfaces?.some(i => i.name === `Node`)
+
+      const isAListOfGatsbyNodeInterfaces =
+        (field.type.kind === `LIST` || field.type.ofType?.kind === `LIST`) &&
+        implementsNodeInterface
+
+      return isAListOfGatsbyNodeInterfaces
+    },
+
+    transform: transformListOfGatsbyNodes,
+  },
+
```

Comment:
Many (or all?) connection field types are now interfaces, so we need this new type transformer to account for that